### ISSUE TITLE
subfindhdf is not included in version 1.2.0

### ISSUE THREAD

**Issue by rainwoodman** [sbird/fake_spectra#11]
```
~/anaconda3/install/lib/python3.6/site-packages/fake_spectra/halocat.py in <module>()
      3 
      4 import numpy as np
----> 5 import subfindhdf
      6 try:
      7     xrange(1)

ModuleNotFoundError: No module named 'subfindhdf'
```

**Comment by sbird** [comment]
Oops.

**Comment by sbird** [comment]
Should be fixed in version 1.2.1: the file was actually included, but the import was wrong.
